docs/docs/resources/pipeline-components/kafka-sink-connector.yaml

```diff
@@ -0,0 +1,47 @@
+# Kafka sink connector
+- type: kafka-sink-connector
+  name: kafka-sink-connector # required, clean up jobs for the connector will run
+  namespace: namespace # required, clean up jobs for the connector will run here
+  # `app` contains application-specific settings, hence it does not have a rigid
+  # structure. The fields below are just an example. Extensive documentation on
+  # sink connectors: https://kafka.apache.org/documentation/#sinkconnectconfigs
+  app: # required
+    tasks.max: 1
+    replication.factor: -1
```

Comment:
replication factor does not make much sense here

Reply:
I just removed it